javascript/javascript3/week1/giphy-api.js

```diff
@@ -0,0 +1,25 @@
+const gifKey = "t2bqf6dFtQjkVxvGZj9ldvsQ2Q0k1Gjp";
```

Comment:
Keep API keys in a seperate JSON file that you add to your .gitignore file and import into this.